cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -180,7 +180,8 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
         names.add(name);
       }
     }
-    responder.sendJson(HttpResponseStatus.OK, applicationLifecycleService.getApps(namespace, names, artifactVersion));
+    responder.sendJson(HttpResponseStatus.OK, applicationLifecycleService.getApps(namespace.toEntityId(),
```

Comment:
oh I see. its the return type of ``validateNamespace``. Probably better to just ignore this comment for this PR.